contracts/TokenHolder.sol

```diff
@@ -361,6 +361,11 @@ contract TokenHolder is MultiSigWallet {
         private
         returns (bytes32 messageHash_, address key_)
     {
+        require(
+            _to != address(this),
+            "Target of a transaction cannot be TokenHolder itself."
+        );
+
```

Comment:
The branch is not rebased with latest develop changes.
We should rebase with latest develop. Require for EIP20 token has been put at the start of executeRule function. I think we should put both requires together after rebasing with develop.

Reply:
✅